### DISCUSSION TITLE
Searching across maps is a great feature

### DISCUSSION THREAD

**Discussion by Olifair** [freeplane/freeplane#1790] [Ideas]
Hello, 
I have been using Freeplane for years, and am still discovering functionality :-) Like yesterday, when I wanted to search for a keyword to get back to some information that I had noted, but didn't quite remember in which map it would be..... and then I thought "searching across open maps would be great" and then I discovered that it was already implemented :-)

So that was a great discovery. 

However then I realised that I would have liked to be looking for my keyword in the node text, but that the find-across-o-maps is looking everywhere, and found the keyword in many comments ... 

So this function would be even better if one could choose if the search should be done in the nodes or in the comments :-)

Cheers

Oliver

**Comment by Tetsuosp** [comment]
Hello,

Could you use this "search" functionality accros map who are not "open" ?

And then how ?

That's the point i need for long time ago because i write my every notes in a map, store them in different folder (project, team manage, private live, etc ...) but finding "one" thing could be difficult ...

Great regards to you all and a happy new year :-)
